src/m365/planner/commands/plan/plan-details-get.ts

```diff
@@ -117,25 +111,30 @@ class PlannerPlanDetailsGetCommand extends GraphItemsListCommand<any> {
       });    
   }
 
-  private getPlanDetails(args: CommandArgs): Promise<any> {
+  private getPlanDetails(logger: Logger,args: CommandArgs,cb: () => void): void {
     const requestOptions: any = {
       url: `${this.resource}/v1.0/planner/plans/${args.options.planId}/details`,
       headers: {
-        'accept': 'application/json'
+        'accept': 'application/json;odata.metadata=none'
       },
       responseType: 'json'
     };
 
-    return request.get(requestOptions);
+    request
+      .get<{ value: PlanDetails }>(requestOptions)
+      .then((res: any): void => {
```

Comment:
Since we have typed the data in the request, let's use the same type here, instead of `any`

Reply:
done